rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/AbstractOperationService.java

```diff
@@ -65,8 +66,15 @@ public Set<OperationDetail> getSupportedOperationDetails(final boolean includeUn
         } else {
             operationClasses = getSupportedOperations();
         }
-        Set<OperationDetail> operationDetails = new HashSet<>();
-
+        Set<OperationDetail> operationDetails = new TreeSet<>((operationDetail1, operationDetail2) -> {
+            try {
+                String simpleName = Class.forName(operationDetail1.getName()).asSubclass(Operation.class).getSimpleName();
+                String simpleName1 = Class.forName(operationDetail2.getName()).asSubclass(Operation.class).getSimpleName();
+                return simpleName.compareTo(simpleName1);
```

Comment:
This would be clearer:
```suggestion
                String simpleName1 = Class.forName(operationDetail1.getName()).asSubclass(Operation.class).getSimpleName();
                String simpleName2 = Class.forName(operationDetail2.getName()).asSubclass(Operation.class).getSimpleName();
                return simpleName1.compareTo(simpleName2);
```